Makefile

```diff
@@ -0,0 +1,43 @@
+PREFIX := /usr/local
```

Comment:
if this needs to get more complicated, maybe lets look in to make alternatives. https://github.com/go-task/task and https://getgb.io/ look decent

Reply:
Thanks for sharing! Will keep this in mind as this project evolves.